package.json

```diff
@@ -9,7 +9,8 @@
     "integration": "nyc --reporter=lcov --reporter=text-lcov -s --all mocha 'test/integration/**/*.js'",
     "post-integration": "nyc report --reporter=lcov > coverage.lcov && codecov -F integration && rm -rf ~/.autolabjs",
     "feature": "nyc --reporter=lcov --reporter=text-lcov -s --all cucumber-js 'test/feature/features' -r 'test/feature/steps'",
-    "post-feature": "nyc report --reporter=lcov > coverage.lcov && codecov -F feature"
+    "post-feature": "nyc report --reporter=lcov > coverage.lcov && codecov -F feature",
+    "quality" : "eslint . & jsinspect . & buddy ."
```

Comment:
now the style is to put a comma even on the last line so that the diff shows modification only on one line.

Reply:
"quality" : "eslint . && jsinspect . && buddy ."

Reply:
Sir, if `eslint` fails the none of the further checks are taken, i.e., none of jsinspect/buddy commands are executed.

Reply:
Also sir, `json` doesn't accept a trailing comma. Throws the following error :
```
npm ERR! JSON.parse Unexpected token } in JSON at position 831 while parsing near '...ect . & buddy .",
npm ERR! JSON.parse   },
npm ERR! JSON.parse   "bin": {
npm ERR! JSON.parse     "a...'
npm ERR! JSON.parse Failed to parse package.json data.
npm ERR! JSON.parse package.json must be actual JSON, not just JavaScript.
```